lib/rubocop/cops_documentation_generator.rb

```diff
@@ -158,8 +158,8 @@ def cop_subsection(title, cop)
   end
 
   def example_header(title, cop)
-    content = "[##{to_anchor(title)}-#{to_anchor(cop.cop_name)}]\n"
-    content << +"==== #{title}\n"
+    content = +"[##{to_anchor(title)}-#{to_anchor(cop.cop_name)}]\n"
```

Comment:
This is the gist of it. Interpolated strings are frozen on ruby 2.7, following the magic literal comment.